### PR TITLE
Fix path state section and structure of sections

### PR DESCRIPTION
**Pull request by mirjak**

### CODE REVIEW

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -674,33 +673,25 @@ packet number space.
...
 
 A path in the "Validating" state performs path validation as described
-in {{Section 8.2 of QUIC-TRANSPORT}}.
+in Section {{path-initiation}}.
 
 The endpoint can use all paths in the "Active" state, provided
 that the congestion control and flow control currently allow sending
-of new data on a path. Note that if a path became idle due to a timeout,
-the endpoint SHOULD send a PATH_ABANDON frame before closing the path.
+of new data on a path.
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:686]:
```suggestion
of new data on a path.
```

File: draft-ietf-quic-multipath.md [hunk 2]
```diff
@@ -517,16 +517,50 @@ the server MAY wait for a short, limited time such as one PTO if a path
 probing packet is received on a new path before sending the
 CONNECTION_CLOSE frame.
 
+### Idle Timeout {#idle-time-close}
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:520]: I moved this section up as this section is supposed to be a subsection of "[Path Close](https://quicwg.org/multipath/draft-ietf-quic-multipath.html#name-path-close)" while the other sections now afterwards are not supposed be subsections of "Path Close".

The text was only moved; no text changes!

File: draft-ietf-quic-multipath.md [hunk 3]
```diff
@@ -661,7 +660,7 @@ after a spurious estimate of path abandonment by the client.
 ~~~
 {: #fig-path-states title="States of a path"}
 
-In non-final states, hosts have to track the following information.
+In all bit the "Closed" states, hosts have to track the following information.
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:663]:
```suggestion
In all but the "Closed" states, hosts have to track the following information.
```

**Review by Yanmei-Liu — APPROVED** [review]

**Review by huitema — APPROVED** [review]
The editorial decision to move the section up is fine. I think we still have issues with the idle timeout procedure, which may lead to ambiguous situations in which the server has dropped resource for one path but the client has not. I opened issue #366 to discuss that.

**Pull request merged**